### ISSUE TITLE
Node version support

### ISSUE THREAD

**Issue by MPParsley** [zazuko/ontology-manager#42]
**Trifid doesn't support Node >12, for this reason the Ontology Manager requires Node 12.**

source: https://github.com/zazuko/ontology-manager/blame/master/docs/local-development-setup.md#L10

Is this still the case?

**Comment by ktk** [comment]
Yes, right now this is still the case. Keep an eye on https://github.com/zazuko/trifid/issues/83, this needs to be resolved.
